lib/gdpr/use_case/delete_users.rb

```diff
@@ -0,0 +1,11 @@
+class Gdpr::UseCase::DeleteUsers
```

Comment:
I'd call this `DeleteStaleUsers` or `DeleteOldUsers` - something like that?

Reply:
Fair enough, called it DeleteInactiveUsers